app/scripts/components/common/cookie-consent/index.scss

```diff
@@ -0,0 +1,24 @@
+.modal {
```

Comment:
this style is breaking the style of other modals : ex. https://deploy-preview-1127--veda-ui.netlify.app/exploration?datasets=%5B%5D&taxonomy=%7B%7D

There are many ways to resolve this problem, but I wonder if this needs to be modal at all. (Considering modal usually includes the blocking of the rest of the screen.) What do you think? Was there any other component you considered using ?

Reply:
Oooh good catch! No there is no need for this to be named `modal` any longer I think that was just a naming convention issue. I have gone ahead and swapped the implementation to `id=cookie-consent`. As for other component we did look at a modal but out of the box the modals are very disruptive to the user experience as a whole and Fausto and I decided against pursuing that direction.